tests/unit/mapperTest.js

```diff
@@ -0,0 +1,43 @@
+'use strict'
+
+const tape = require('tape')
+const { createMappedObject } = require('../../src/middleware/mapper')
+
+
+tape.test('Mapper', t => {
+  t.test('createMappedObject()', t => {
+    t.test('should throw when mapping schema is not supplied', t => {
+      t.throws(createMappedObject, new Error(`No mapping schema supplied`))
+      t.end()
+    })      
+
+    t.test('should create an object based on mapping', t => {
+      const ctx = {
+        request: {
+          body: {
+            inputOne: 1, 
+            inputTwo: 2, 
+            inputThree: 3
+          } 
+        }
+      }
+
+      const mappingSchema = {
+        'inputOne': 'outputOne',
+        'inputTwo': 'outputTwo',
+        'inputThree': 'outputThree'
+      }
+
+      const expected = {
+        outputOne: 1, 
+        outputTwo: 2, 
+        outputThree: 3
+      }
+
+      createMappedObject(ctx, mappingSchema)
+
+      t.equal(JSON.stringify(ctx.body), JSON.stringify(expected))
```

Comment:
I think we could use something like `deepEqual` to check the object actually is the same instead of converting it to string and comparing the string values
https://github.com/substack/tape#tdeepequalactual-expected-msg